core-bundle/src/Resources/public/core.js

```diff
@@ -992,10 +994,53 @@ var Backend =
 	},
 
 	/**
-	 * Get the current scroll offset and store it in a cookie
+	 * Store the current scroll offset in sessionStorage
 	 */
 	getScrollOffset: function() {
-		document.cookie = "BE_PAGE_OFFSET=" + window.getScroll().y + "; path=" + (Contao.path || '/');
+		window.sessionStorage.setItem('contao_backend_offset', parseInt(window.getScroll().y, 10));
```

Comment:
Just FYI, the `window` prefix is not necessary

Reply:
I know.